charters/UNITY/charter.md

```diff
@@ -0,0 +1,54 @@
+# Unity Benchmarking Service
+
+## Description
+
+The Unity benchmarking service automates the process of running and benchmarking pipelines.
```

Comment:
What is a _pipeline_?

Reply:
changed to "data processing pipelines" which matches the language of the other charters.